src/react/components/_finances/FinancialResources/HasAccessTo.js

```diff
@@ -256,7 +256,11 @@ const hasAccessToLink = (key, roles, careers, programs, delegate, summer) => {
     : true;
   const hasPermittedCareer = currentLinkAccess.careers
     ? careers.filter(value => currentLinkAccess.careers.includes(value))
-        .length > 0
+        .length > 0 ||
```

Comment:
@marcpechaitis - Is there some way to make this more readable? Are you using a one-line if statement...but with like multiple lines?

Reply:
That is how Prettier formatted it?

@redconfetti are you asking me to refactor it?

Reply:
@marcpechaitis - I was suggesting you refactor it to be easier to read, but it's fine.